### DISCUSSION TITLE
Error during compile of Emgu CV Open Source Release on macOS

### DISCUSSION THREAD

**Discussion by antoniorotundo2** [emgucv/emgucv#735] [Q&A]
Hi,
I am trying to compile open source release of Emgu CV library on macOS, following these steps:

1. checkout the source:
git clone https://github.com/emgucv/emgucv emgucv

2. go to emgucv folder:
cd emgucv

3. initialize opencv, tesseract-ocr and cvblob submodules
git submodule update --init --recursive

4. install Homebrew, a macOS package manager, in order to satify compilation dependencies
https://brew.sh/
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"

5. Install CMake as cask package, that is, using its installer preferences
https://formulae.brew.sh/cask/cmake
brew install --cask cmake

6. Install Mono as cask package, an Open source implementation of .NET Framework
https://formulae.brew.sh/cask/mono-mdk
brew install --cask mono-mdk

7. Install the dotnet-sdk as cask package
https://formulae.brew.sh/cask/dotnet-sdk
brew install --cask dotnet-sdk

8. Go to the configuration folder
cd platforms/macos

9. Use the following command to configure and build the project, having an Apple Silicon processor
./configure_arm64

10. At the end of execution, give me the following error message
"/Users/antoniorotundo/emgucv/Solution/Mac/Emgu.CV.Mac.Example.sln" (destinazione: Restore) (1) ->
(destinazione: Restore) -> 
  /Library/Frameworks/Mono.framework/Versions/6.12.0/lib/mono/msbuild/Current/bin/NuGet.targets(131,5): error : Package Microsoft.VisualStudio.Web.CodeGeneration.Design 6.0.2 is not compatible with net60 (.NETFramework,Version=v6.0). Package Microsoft.VisualStudio.Web.CodeGeneration.Design 6.0.2 supports: net6.0 (.NETCoreApp,Version=v6.0) [/Users/antoniorotundo/emgucv/Solution/Mac/Emgu.CV.Mac.Example.sln]

Can anyone help me solve the problem? I am attaching the complete log, generated during the compilation of the library, if it can be useful to understand better the question.
[configure_macos_log.txt](https://github.com/emgucv/emgucv/files/8903753/configure_macos_log.txt)

**Comment by emgucv** [comment]
In this case, one of the demo project requires ASP.net.  

I would recommand installing Visual Studio for Mac if you want to compile Emgu CV from source. The latest release of VS supports Apple Silicon.
